File: samps/src/isumit.py
#
# File:    isumit.py
# Purpose: to sum the integers from stdin
# Author:  pc2@ecs.csus.edu or http://www.ecs.csus.edu/pc2
#
# $Id: sumit.pl 326 2006-10-25 02:53:57Z laned $
#

import sys

# softspace required to avoid extra space after print
# sys.stdout.softspace=0

hand = sys.stdin

sum = 0

for line in hand .readlines():
  num = int(line);
  if num > 0:
    sum += num

print("The sum of the integers is %s"% sum)

# eof sumit.py $Id: hello.py 2323 2011-09-11 02:02:34Z laned $

File: samps/src/isumit.kt

//
// File:    isumit.kt
// Purpose: to sum the positive integers from stdin
// Author:  pc2@ecs.csus.edu or http://www.ecs.csus.edu/pc2
//

fun main(args: Array<String>) {
   var sum : Int = 0

   var T = -1
   while (T != 0) {
      T = readLine()!!.toInt()
      if (T > 0) {
         sum += T
      }
   }
   print("The sum of the integers is $sum\n")
}
